beacon-chain/node/node.go

```diff
@@ -226,6 +226,7 @@ func (b *BeaconNode) registerP2P(ctx *cli.Context) error {
 		StaticPeers:       sliceutil.SplitCommaSeparated(ctx.GlobalStringSlice(cmd.StaticPeers.Name)),
 		BootstrapNodeAddr: bootnodeENR,
 		RelayNodeAddr:     ctx.GlobalString(cmd.RelayNode.Name),
+		BaseDataDir:       ctx.GlobalString(cmd.DataDirFlag.Name),
```

Comment:
What does "base" data imply here? Maybe "p2p" data or "network" data? Better to be explicit

Reply:
Why not call it DataDir?

Reply:
Resolved